app/models/katello/repository.rb

```diff
@@ -306,7 +306,7 @@ def self.clone_docker_repo_path(options)
                                            :content_view_id => options[:content_view]).first
         view = repo.content_view.label
         product = repo.product.label
-        env, _ = cve.label.split('/')
+        env, = cve.label.split('/')
```

Comment:
Yea that `env, =` looks weird, I don't like it but you can avoid such a case with `env = cve.label.split('/').first`, which I think is more readable